.github/actions/rockylinux-build/entrypoint.sh

```diff
@@ -7,10 +7,10 @@ fi
 
 DEST_DIR=$GITHUB_WORKSPACE/rpms
 
-echo "Copying prebuilt RPMs for $IMAGEMAGICK_VERSION to $DEST_DIR"
+echo "Copying prebuilt packages for $IMAGEMAGICK_VERSION to $DEST_DIR"
 mkdir -p "$DEST_DIR"
-cp /root/rpmbuild/RPMS/x86_64/*.rpm "$DEST_DIR"
+cp /root/rpmbuild/RPMS/x86_64/*.rpm  "$DEST_DIR"
```

Comment:
```suggestion
cp /root/rpmbuild/RPMS/x86_64/*.rpm "$DEST_DIR"
```

Reply:
always check for unwanted changes in the PR diffs - I do for my own before asking for review usually